modules/electrophysiology_browser/php/electrophysiologybrowserrowprovisioner.class.inc

```diff
@@ -47,10 +47,17 @@ class ElectrophysiologyBrowserRowProvisioner
               s.Visit_label                             AS Visit_Label,
               MIN(pf.AcquisitionTime)                   AS Acquisition_Time,
               MIN(pf.InsertTime)                        AS Insert_Time,
-              GROUP_CONCAT(DISTINCT pot.OutputTypeName) AS Links,
+              GROUP_CONCAT(DISTINCT CASE 
+                WHEN pot.OutputTypeName='derivatives' THEN 'raw,derivative'
+                ELSE 'raw'
```

Comment:
This works until a new type is added in OutputTypeName. It will fall in the else case and be assigned to 'raw'.